libs/utils/utils.js

```diff
@@ -840,7 +840,12 @@ async function loadPostLCP(config) {
 export function scrollToHashedElement(hash) {
   if (!hash) return;
   const elementId = hash.slice(1);
-  const targetElement = document.querySelector(`#${elementId}:not(.dialog-modal)`);
+  let targetElement;
+  try {
+    targetElement = document.querySelector(`#${elementId}:not(.dialog-modal)`);
+  } catch (e) {
+    window.lana?.log(`Cannot make URL from metadata - ${elementId}: ${e.toString()}`);
```

Comment:
I don't exactly get why this is tied to metadata, are they related somehow? Also, I don't see a URL being created in the `try` block, I think this message might not be accurate.

Reply:
Maybe we can change the message to "Could not query element based on hash"

Reply:
Copy paste error, changed it then had to re-do the branch and those changes got lost.. will fix it!